src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/TextBlockControl/SimpleText_MaxLines_Mutiple_Containers.xaml.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Uno.UI.Samples.Controls;
+using Windows.UI.Xaml.Controls;
+
+
+namespace Uno.UI.Samples.Content.UITests.TextBlockControl
+{
+	[SampleControlInfo("TextBlockControl", "SimpleText_MaxLines_Mutiple_Containers", description: "This sample shows two different parents for a text block. The maximum number of lines they should have is defined by the slider.")]
```

Comment:
New approach is to use simply `[Sample]` attribute. The name & category are infered from class name/namespace.